org.lflang/src/org/lflang/federated/FederateInstance.xtend

```diff
@@ -389,6 +353,84 @@ class FederateInstance {
         return !excludeReactions.contains(reaction)
     }
     
+    /** 
+     * FIXME: This redundant function is here only because it is used in
+     * another repo (for the TS target) and it is a royal pain to refactor
```

Comment:
I'm confused about this comment. What code/repo is this comment referring to? The `reactor-ts` submodule only has TypeScript code in it...

Reply:
This function has been renamed "contains" from "containsReaction", so this is here just for backward compability in another repo.  The rename has to be done in the other repo, and then everything aligned. I don't have the bandwidth to do this.  I've added a more detailed explanation.

Reply:
I still don't understand. What repo are we talking about? Can you provide a link?

Reply:
If the method is here only to preserve compatibility, it should be labeled with `@Deprecated`. However, I also don't see how the TS runtime uses this method.